.github/labeler.yml

```diff
@@ -0,0 +1,27 @@
+version: 1
+labels:
+  - label: 'tag: new feature'
+    title: '^feat(\(.+\))?:.*'
+
+  - label: 'tag: bug fix'
+    title: '^fix(\(.+\))?:.*'
+
+  - label: 'tag: polish'
+    title: '^refactor(\(.+\))?:.*'
```

Comment:
It's a bit complicated, but if you check the changelog, `feat` can be either "new feature" or "polish", and `fix` can be either "bug fix" or "polish" as well😅 `refactor`, on the other hand, can often be "maintenance" or "internal".

Our tags system is orthogonal to the semantic messages, and that's more or less intentional. I'm not against this change but just to illustrate the complication.

Reply:
Yes, but in most cases it should make work on manual labeling easier, at least a little.

Reply:
That's true